qa-include/db/install.php

```diff
@@ -67,6 +67,11 @@ function qa_db_table_definitions()
 	require_once QA_INCLUDE_DIR . 'db/maxima.php';
 	require_once QA_INCLUDE_DIR . 'app/users.php';
 
+	if (defined('QA_USE_UTF8MB4') && QA_USE_UTF8MB4)
```

Comment:
As this same check is used many times, I'd suggest extracting it into a function e.g. `qa_db_get_mysql_collation()`